docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+by the absolute value of -29.4%. All thresholds are customizable through the API of the
+:class:`~shift_detector.checks.simple_check.SimpleCheck`. All the metrics are listed in the section
+:ref:`simple_check_parameters`.
+
+Categorical
+~~~~~~~~~~~
+
+Using the SimpleCheck shifts can also be detected on categorical columns. Here the histograms over the attribute-values are
+compared. If the difference between those values exceeds the *categorical_threshold* the check calls shift on this
+column.
+
+The example above shows a shift in the categorical column 'payment_option'. There are different
+attribute-values in this column, i.e. 'debit and 'cash'. In those, the differences between the dataset are 6% and
+-6.5% which both exceed the threshold of 5%. This indicates that the in dataset2 more people use
+cash as a payment-option and fewer use debit, the check calls shift on the column.
+
+
+.. _simple_check_parameters:
+
+Metrics & Parameters
+----------
+
+There are 9 different numerical metrics in the SimpleCheck whose differences can indicate a shift. All
+default-thresholds can be adjusted.
+
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| metric_name                       |                                                                           explanation                                                   |
++===================================+=========================================================================================================================================+
+| **quartile_1,                     |                                                                                                                                         |
+| median,                           | Those are the .25- .5- and .75-quantiles of the column.                                                                                 |
+| quartile_3**                      |                                                                                                                                         |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| **value-range**                   | The range of values of the columns, calculated as maximum minus minimum of that column.                                                 |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| **std**                           | the standard deviation in the column                                                                                                    |
```

Comment:
std seems a little bit vacuous. Can we rename it to `stddev`?

Reply:
its commonly used like that see https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.std.html